CondFormats/HcalObjects/interface/HcalMahiConditionsHost.h

```diff
@@ -0,0 +1,11 @@
+#ifndef CondFormats_HcalObjects_interface_HcalMahiConditionsHost_h
+#define CondFormats_HcalObjects_interface_HcalMahiConditionsHost_h
+
+#include "CondFormats/HcalObjects/interface/HcalMahiConditionsSoA.h"
+#include "DataFormats/Portable/interface/PortableHostCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
```

Comment:
I think these two are not needed here ?
```suggestion
```

Reply:
Done in 4615e8825352834ac1dd4f1bbe766b0cafc1d321